networkRequests/roles.js

```diff
@@ -0,0 +1,41 @@
+import API_URL from "../components/globalApiUrl";
+
+export const roleChange = async (userID, role) => {
+  return fetch(API_URL + "/roleschange", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      token:
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1YzhjOTFlZDA1MzdmZTBkNDlmMTU2ZWIiLCJoYXNoZWRQYXNzd29yZCI6Im5pdHMiLCJpYXQiOjE1NTI3MTY1OTQsImV4cCI6MTU1MjgwMjk5NH0.Necq4MXl_7L5XpFPzTbdW-AGxlpcYDVZBGr88kFD6WM",
+      userID: "5c8c91a7aa68cb0d3dae645a"
+    },
+    body: {
+      userID,
+      role
+    }
+  })
+    .then(resp => {
```

Comment:
you can use the async await syntax to make it clearer and more readable. So instead you would ``` const resp = await fetch(...)``` and then be able to use that as a normal variable later on